docs/pages/guides/customizing-webpack.mdx

```diff
@@ -45,39 +45,6 @@ module.exports = async function (env, argv) {
 };
 ```
 
-## Polyfills
```

Comment:
ResizeObserver has 95% global browser usage, so I dropped this section. It's not strictly related to the entire PR.